app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,81 @@
+<h1 class="w-fit text-3xl font-semibold ml-[40%] my-6 underline underline-offset-1"><%= @recipe.name %></h1>
+<div class="w-4/5 ml-[5%]">
+  <div class="flex justify-between items-center">
+    <span class="text-2xl font-medium">Preparation time: <%= @recipe.preparation_time %> minutes</span>
+    <% if @recipe.user == current_user %>
+      <%= form_with model:@recipe, url: recipes_path  do |form| %>
+        <label class="relative inline-flex items-center cursor-pointer">
+          <%= form.check_box :public, class: 'sr-only peer', id: 'public-toggle' %>
+          <div class="w-11 h-6 bg-gray-200 peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-red-300 rounded-full peer peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all peer-checked:bg-red-600"></div>
+          <span class="ml-3 text-sm font-semibold flex items-center">Public</span>
+        </label>
+      <% end %>
+    <% end %>
+  </div>
+  <span class="text-2xl font-medium">Cooking time: <%= @recipe.cooking_time %> minutes</span>
+  <div class="flex items-center justify-between mt-4">
+    <p class="text-lg"><%= @recipe.description %></p>
+    <% if @recipe.user == current_user %>
+      <%= link_to 'Add ingredient', new_user_recipe_recipe_food_path(recipe_id: @recipe, user_id: @recipe.user), class:"bg-yellow-300 py-2 px-4 rounded-full" %>
+    <% end %>
+  </div>
```

Comment:
- [ ] KIndly take a look again at the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md), it seems like you are missing out on a `General Shopping List` button in this section of the code. :hammer_and_wrench: